### PR TITLE
Replace anchors with 'contain'

### PR DESCRIPTION
**Pull request by wyardley**
replace anchors with `contain`

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -33,10 +33,9 @@
   include '::nodejs::install'
 
   if $manage_package_repo {
-    include $repo_class
-    anchor { '::nodejs::begin': }
-    -> Class[$repo_class]
+    contain $repo_class
```
Comment (ekohl) [inline manifests/init.pp]: Including it is better. Say a user has ```Class['::epel'] -> Class['::x'] -> Class['::nodejs']``` then trying to ```contain ::epel`` here will lead to issues while it shouldn't be. If the installation is contained and you set the repo class as a dependency then you're set. We don't care where epel is (as long as it's before us) and we don't give a guarantee about it.

File: manifests/init.pp [hunk 2]
```diff
@@ -33,10 +33,9 @@
   include '::nodejs::install'
```
Comment (ekohl) [inline manifests/init.pp]: I would contain this.

File: manifests/init.pp (hunk 1 above)
Comment (wyardley) [reply manifests/init.pp]: What was the anchor class around it for before then? Is it not needed, even if we switch back to `include`?

Comment (wyardley) [reply manifests/init.pp]: See if the version I just pushed looks better

Comment (ekohl) [reply manifests/init.pp]: I think the previous implementation was incorrect and this is the correct one.

**Pull request merged**